lib/QubitXmlImport.class.php

```diff
@@ -414,7 +418,7 @@ private function populateObject(&$domNode, &$importDOM, &$mapping, &$currentObje
 
     $doSave = true;
     // if this is an information object in an XML EAD import, run the enhanced matching check.
-    if ($currentObject instanceof QubitInformationObject && $importSchema == 'ead')
+    if ($currentObject instanceof QubitInformationObject && $importSchema == 'ead' &&  $this->options['match'])
```

Comment:
If the key 'match' isn't in $this->options I think it will throw a PHP notice or warning (sometimes can't see them without error_reporting(E_ALL) :/). I recommend using ```isset($this->options['match']) && $this->options['match']```